src/com/facebook/buck/swift/SwiftLibraryDescription.java

```diff
@@ -332,6 +353,7 @@ public static boolean isSwiftTarget(BuildTarget buildTarget) {
     public Optional<SourcePath> bridgingHeader;
     public Optional<ImmutableSortedSet<BuildTarget>> deps;
     public Optional<NativeLinkable.Linkage> preferredLinkage;
+    public Optional<ImmutableMap<Path, SourcePath>> exportedHeaders;
```

Comment:
Need to update `SwiftLibraryIntegrationTest` as well to set `args.exportedHeaders` to `Optional.absent()`